CMakePresets.json

```diff
@@ -106,20 +106,20 @@
         {
             "name": "ci-ninja-debug",
             "inherits": [
-                "ci-ninja",
-                "clang-tidy"
+                "ci-ninja"
             ],
             "displayName": "Ninja Debug",
             "cacheVariables": {
                 "CMAKE_BUILD_TYPE": "Debug"
             }
         },
         {
-            "name": "ci-ninja-debug-notidy",
+            "name": "ci-ninja-debug-tidy",
```

Comment:
Instead of disabling clang-tidy in default compilation, I think we can disable clang-tidy at the per-target level (e.g. for only unity build or pch target)